### PR TITLE
added attachments

### PR DESCRIPTION
**Pull request by nerko937**

### CODE REVIEW

File: example_project/urls.py [hunk 1]
```diff
@@ -13,12 +13,15 @@
...
 from django.urls import path, include
 
+
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('auth/', include('dj_rest_auth.urls')),
     path('users/', include('users.urls')),
     path('projects/', include('projects.urls')),
-]
+] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```
Comment (robertmusil) [inline example_project/urls.py:27]: why do you add this to urls?

File: projects/views.py [hunk 2]
```diff
@@ -33,3 +33,22 @@ class IssueListForProject(generics.ListAPIView):
...
         return Issue.objects.filter(project_id=self.kwargs['project_id'])
+
+
+class AttachmentCreate(generics.CreateAPIView):
+    serializer_class = AttachmentSerializer
+    permission_classes = (IsAuthenticated,)
+
+
+class AttachmentDestroy(generics.DestroyAPIView):
+    queryset = Attachment.objects.all()
```
Comment (robertmusil) [inline projects/views.py:44]: i don't understand the queryset here, what's the point?

**Review by robertmusil — APPROVED** [review]

File: projects/views.py (hunk 2 above)
Comment (nerko937) [reply projects/views.py:44]: When i delete it, it throws error:
```
AssertionError at /projects/attachments/2/

'AttachmentDestroy' should either include a `queryset` attribute, or override the `get_queryset()` method.
```

File: example_project/urls.py (hunk 1 above)
Comment (nerko937) [reply example_project/urls.py:27]: to expose links to files, without it drf navigates to file where is 404 instead

**Pull request merged**